src/project-management/release-train-model.md

```diff
@@ -0,0 +1,54 @@
+# Release Train Model
+
+## What is release train model?
+
+Before v5.0, TiDB releases a major version every year or every half year.
+
+However, the longer a development cycle is, the more features it is responsible to deliver, which is hard to coordinate when doing regression and acceptance tests and releasing, and thus delay happens.
+
+Also, new feature requests occur during the development cycle, and due to the long development cycle, new feature requests are added as development backlog after the development cycle started. In this case, development tasks can hardly converge until it is closed to release date.
+
+Starting from v5.0, TiDB adopts the release train model. Release train model is a product development model for requirements gathering, analysis, decision making, release, and issue feedback.
+
+Just like a train delivering goods, decisions need to be made about the priority of the goods, destination, arrival time, which train to load on, which carriage, etc., before the train departs.
+
+The benefits of moving to the release train model are:
+
+1. A quicker feedback cycle and users can benefit from features shipped quicker.
+2. Predictability for contributors and users:
+    1. Developers and reviewers can decide in advance what release they are aiming for with specific features.
+    2. If a feature misses a release train we have a good idea of when it will show up.
+    3. Users know when to expect their features
+3. Transparency. There will be a published cut-off date (AKA code freeze) for the release and people will know about it in advance. Hopefully this will remove the contention around which features make it.
```

Comment:
Meaning changes. The original expression focuses that we have code freeze so that features miss the date won't be included. "the contention about which features can meet the date" expresses weirdly.

Reply:
Similar comment here, one common tip to make the content as clear as possible is using pronouns carefully. For the last sentence, perhaps, we could make it clear that what "this" and "it" represents to avoid confusion and misunderstanding.

Reply:
see https://github.com/pingcap/tidb-dev-guide/pull/60/files#r676745372